### ISSUE TITLE
Focus Not Obscured (minimum) should not prohibit cookie popups

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#2551]
As written, the new [WCAG 2.1 success criterion 2.4.12: Focus Not Obscured (Minimum)](https://w3c.github.io/wcag/understanding/focus-not-obscured-minimum.html) has no exception for obscuring content that can be dismissed. That includes most cookie popups, which are generally implemented as fixed position bars at the bottom of the viewport. Because of that design, most of the ones I've looked at do obscure the focus at on point or another in navigating that page.

While that is certainly a problem, it seems pretty reasonable to expect users who needs that popup to disappear, to use their dismiss feature to get rid of it. That can be improved even further by ensuring quick keyboard access to the cookie banner (skiplink, or tabindex=1).

**Comment by WilcoFiers** [comment]
Also to clarify my position on this. I would be in favour of an exception, but I do not object to the SC without one. In either case the understanding document should explicitly call out obscuring content that can be dismissed, and use the cookie popup as an example.

**Comment by WestonHanners** [comment]
My understanding of the SC is that cookie popups are allowed, but the padding needs to be adjusted underneath so that the user cannot focus on obscured items. If a cookie popup doesn't consider this, I would imagine it should be corrected rather than exempted.

A fully blind user might not understand that the content is obscured till they try to interact with it. Then, locating the obscuring element could be troublesome.

**Comment by alastc** [comment]
> most of the ones I've looked at do obscure the focus at on point or another in navigating that page

Indeed, they are a good example of the type of thing this SC is supposed to catch.

For the understanding doc I suggest that we take an angle such as: 

"If there is a mandatory banner implemented as sticky content (e.g. a cookie banner at the bottom of the viewport), the website must have a mode of operation where the focus is not obscured by the cookie banner. For example, the banner is first in the focus order so it can be dismissed before navigating through the page. Other options include making the banner modal so the user has to dismiss the banner before navigating through the page, or to use the (link) scroll padding technique so there is no overlap."

Does that seem reasonable?

Anecdotally, the ones I've seen that claim to be accessible don't suffer from this problem, e.g. https://www.iicsa.org.uk/

**Comment by WestonHanners** [comment]
I agree with this and the iicsa.org link you shared is actually pretty good and probably the appropriate action for AA, I think achieving AAA should probably not obscure the full page with a transparent layer that cannot be clicked through. Semi-transparent modals (which is what this site has) are still somewhat problematic.

**Comment by mbgower** [comment]
I've updated @fstrr's PR based on discussion at Tuesday's AGWG call.
